Plugins/Alpakit/Alpakit.uplugin

```diff
@@ -1,16 +1,18 @@
 {
 	"FileVersion": 3,
 	"Version": 1,
-	"VersionName": "1.0",
+	"VersionName": "2.0.0",
```

Comment:
VersionName major must match Version. You should probably bump Version field value to 2.